lib/info-extras.js

```diff
@@ -124,11 +124,37 @@ exports.getPublished = (body) => {
  * @return {Array.<Object>}
  */
 exports.getRelatedVideos = (body) => {
-  let jsonStr = util.between(body, '\'RELATED_PLAYER_ARGS\': {"rvs":', '},');
+  let jsonStr = util.between(body, '\'RELATED_PLAYER_ARGS\': ', '\n');
+  let watchNextJson;
   try {
-    jsonStr = JSON.parse(jsonStr);
-  } catch (err) {
-    return [];
+    jsonStr = jsonStr.substring(0, jsonStr.length - 1)
```

Comment:
not needed since the last character is a `\n`

Reply:
There is a ' , ' at the end that causes JSON.parse to throw an exception 
![image](https://user-images.githubusercontent.com/10515204/64063500-69527500-cc12-11e9-8c88-012e768984dc.png)